### DISCUSSION TITLE
Three ideas for the selected node display type.

### DISCUSSION THREAD

**Discussion by aaa1386** [freeplane/freeplane#1503] [Ideas]
Freeplane software has two modes for the selected nodes.

First mode: turns the background of the selected node blue.
  If we have defined several styles that have different backgrounds, when we select a node, it is difficult to recognize the selected node.

   Also, if we choose a node with a color background, the color of that node will change to blue, and the main color of that node is not clear, we can tell what color it is when we leave that node.

Which node do you think is selected in the figure below?

![image](https://github.com/freeplane/freeplane/assets/38586259/37770355-572a-4f87-9b2d-e686bdd32a0d)

The second mode: the rectangular selection mode for the selected node.
In this mode, the nodes with colored background do not interfere, but it is difficult to recognize the selected node among the large number of nodes that can be seen on the screen.

  Two ideas for the selected node:
1- The crosshatched mode without changing the color of the node background.
![image](https://github.com/freeplane/freeplane/assets/38586259/4c310a01-d175-4161-9245-9b7750e66704)

2-The color line should be thicker in rectangular selection mode.
3- A combination of the above two modes, which in my opinion is the best mode. The node background is not tampered with. It becomes easy to recognize the selected node among many nodes on the screen.

**Comment by euu2021** [comment]
The Spotlight also helps. I use a purple bubble + Spotlight:
![image](https://github.com/freeplane/freeplane/assets/77707706/06f85dd4-2832-4e69-935f-c042d5e1f3bb)

> First mode: turns the background of the selected node blue.

About this first mode, you know you can set it in the Styles Editor, right? So, you can choose to change a specific property, like the node border or node edge. So, if you don't want it to change the background color, you can choose another property:

![image](https://github.com/freeplane/freeplane/assets/77707706/27435c97-a3e9-4ccf-bcb8-91f04f187a69)

> The crosshatched mode without changing the color of the node background.

This is an interesting idea.

**Comment by aaa1386** [comment]
@euu2021 So, if you don't want it to change the background color, you can choose another property....

If we change the background of the selected node to white, there is still a problem. Because when we select a node that is green for example, the color of that node becomes white!

The problem is solved when there is an option in the software to disable the background of the selected node altogether. As a result, the background of the selected node is the same as the background of that node's style.
To make the selected node clear, let's use the other ways you suggested, for example, I set the border as shown below.

![image](https://github.com/freeplane/freeplane/assets/38586259/42a2b54c-8a66-4a78-8491-bd1c6202e4eb)





https://github.com/freeplane/freeplane/assets/38586259/e962f3fd-6c5b-4f43-8162-95513bf61524

**Comment by aaa1386** [comment]
I think there should be an option where we can choose whether the background of the selected node is active or not.
  If the selected node background is not active, the background of the selected nodes will be displayed in the mindmap with the same base color as their style.

**Comment by aaa1386** [comment]
In any case, thank you for your advice.

**Comment by euu2021** [comment]
> I think there should be an option where we can choose whether the background of the selected node is active or not. If the selected node background is not active, the background of the selected nodes will be displayed in the mindmap with the same base color as their style.

It is possible. Just click the little circle arrow button:

![image](https://github.com/freeplane/freeplane/assets/77707706/1cef0b92-ac3e-4661-9ee9-4bc030048252)

That's how the styles work. If you click that little circle arrow button, the property won't be changed when the style is applied to a node.

**Comment by aaa1386** [comment]
> 
> It is possible. Just click the little circle arrow button:
> 

I am sincerely thankful . It was good advice. It solved my problem.
I have now realized that there is a bug in the software and it would be good if the developers of the software would fix it. .
If there is any life left, I will bring it up at the right time.
